docker-build.sh

```diff
@@ -405,7 +409,7 @@ _build_image_buildkit() {
 
   local cache_to
   if _can_push; then
-    cache_to="--cache-to mode=max,image-manifest=true,type=registry,ref=$cache_image"
+    cache_to="--cache-to mode=max,image-manifest=true,type=registry,compression=zstd,ref=$cache_image"
```

Comment:
I prefer to leave the `compression=zstd` part for a separate (configurable) option as it can increase build time.